src/views/Auction/components/Timer.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react'
```

Comment:
We should a coding convention. We randomly decide to prefix imports like this:

1. `// Externals` for any external module(s)
2. `// Interfaces` for internal interfaces defined in `src/interfaces`
3. `// Components` for Components
4. `// Utils` for any kind of helper functions
5. `// Layouts` for any layout Component defined in `src/layouts`

This is not fixed. I am guilty of deviating away from this. but I am happy to discuss this style further :)

Reply:
yeah that is fine. I have updated my component to fit along these lines.